### PR TITLE
Fix Filters - Return the result after filter is applied

### PR DESCRIPTION
**Pull request by boopathi**

### CODE REVIEW

File: src/main/scala/de/zalando/beard/filter/Filter.scala [hunk 1]
```diff
@@ -66,3 +66,13 @@ object DateFormatFilter {
   def apply(): DateFormatFilter = new DateFormatFilter()
 }
 
+class CapitalizeFilter extends Filter {
```
Comment (danpersa) [inline src/main/scala/de/zalando/beard/filter/Filter.scala]: what about writing an tests for this class? :)

Comment (danpersa) [inline src/main/scala/de/zalando/beard/filter/Filter.scala]: each of the filters should have tests...

Comment (boopathi) [inline src/main/scala/de/zalando/beard/filter/Filter.scala]: done :+1:

File: src/test/scala/de/zalando/beard/filter/FilterSpec.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      case Some(filter) => filter
+      case None => throw FilterNotFound(name)
+    }
+  describe("filters") {
+    it("capitalize") {
+      val filter = getFilter("capitalize")
+      filter.apply("capitalize") should be("Capitalize")
+      filter.apply("cAPITALIZE") should be("CAPITALIZE")
+    }
+    it("uppercase") {
```
Comment (danpersa) [inline src/test/scala/de/zalando/beard/filter/FilterSpec.scala]: the convention for BDD is that the it clauses should be readable... `it ("should upppercase")` in this case

File: src/test/scala/de/zalando/beard/filter/FilterSpec.scala [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package de.zalando.beard.filter
+
+import org.scalatest.{FunSpec, Matchers}
+
+/**
+  * @author boopathi
+  */
+class FilterSpec extends FunSpec with Matchers {
+  def getFilter(name: String) =
```
Comment (danpersa) [inline src/test/scala/de/zalando/beard/filter/FilterSpec.scala]: this method should be private and usually you put the private methods at the end of the file...

**Comment by boopathi** [comment]
Moving capitalize filter and other tests to a different pull request.

This fix is required for filters to work for beard files.

**Pull request merged**